### ISSUE TITLE
Stackdriver Error Reporting requires certain MonitoredResource?

### ISSUE THREAD

**Issue by tazjin** [tazjin/journaldriver#4]
When reporting errors in [the format](https://cloud.google.com/error-reporting/docs/formatting-error-messages) expected by Stackdriver Error Reporting from *outside* of GCP, the errors never show up in the Error Reporting console. This works from a GCP instance running `journaldriver`.

Example log command to reproduce (based on the example from the docs linked above):

```
echo '<3>{"serviceContext":{"service": "worker"},"message":"Test error message","context":{"reportLocation":{"filePath":"cleanup.js","lineNumber":42,"functionName":"processJob"}}}' | systemd-cat
```

Running this on a GCP instance running `journaldriver` will result in Error Reporting picking up the error. Running it from a non-GCP machine (e.g. from my development machine) will only cause the error to show up in the normal log stream, but not in Error Reporting.

My initial theory is that something happens with the [MonitoredResource](https://cloud.google.com/logging/docs/reference/v2/rest/v2/MonitoredResource) attached to a [LogEntry](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry) that determines whether or not Stackdriver Error Reporting will ingest the message.

**Comment by tazjin** [comment]
Here's the list of monitored resources from which errors can be ingested:

>  app_script_function, aws_ec2_instance, cloud_function, container, dataflow_step, gae_app, gce_instance, k8s_container, ml_job, serverless_service, global

(Thanks @saturnism!)

------

Out of these resources the most generic one for ingestion from a non-GCP instance is probably `global`, I'll spend some time reading through the docs to figure out what the best approach is.

**Comment by tazjin** [comment]
We've tested the fix from #9 on non-GCP machines now and it works fine. The issue is of course that we lose the separation into different log-streams, but there is no way around that currently.

I'm closing this issue as we now have a viable way of using Error Reporting.
